app/controllers/new-signup.js

```diff
@@ -46,15 +52,25 @@ export default class NewSignUpController extends Controller {
     this.isButtonDisabled = true;
   }
 
-  @action register() {
+  @action changeStepToFive() {
+    this.currentStep = this.FIFTH_STEP;
     this.analytics.trackEvent(NEW_SIGNUP_FLOW.USER_USERNAME);
     this.isButtonDisabled = true;
+  }
+
+  @action register() {
+    this.analytics.trackEvent(NEW_SIGNUP_FLOW.USER_ROLE);
+    this.isButtonDisabled = true;
     this.signup();
   }
 
   @action completeSignUp() {
     this.analytics.trackEvent(NEW_SIGNUP_FLOW.NEW_SIGNUP_FLOW_DONE);
-    window.open(GOTO_URL, '_self');
+    if (this.isDevMode) {
+      window.open('http://localhost:5500/goto.html?dev=true', '_self');
+    } else {
+      window.open(GOTO_URL, '_self');
+    }
```

Comment:
Why is the URL localhost here?

Reply:
Done with changes...can you please check my latest commit